juju/juju
# Wireup revised secrets model backend

The secrets hook commands and API have been updated to use the revised secrets data model. This PR updates the apiserver and state layers to match. The secrets rotation backend still needs more work in followup PRs.

## Checklist


## QA steps

Check secret create, update, get.
Bootstrap and deploy ubuntu
```
$ juju exec --unit ubuntu/0 "secret-add --label foobar --description blah secret!"
secret:cbm3aqfnuod28tu36h50
$ juju secrets
URI                          Revision  Rotate  Backend  Age
secret:cbm3aqfnuod28tu36h50         1  never   juju     5 seconds ago  
$ juju exec --unit ubuntu/0 "secret-get secret:cbm3aqfnuod28tu36h50"
secret!
```
